src/shared-with-pluot-core/CommonIncludes.js

```diff
@@ -85,6 +85,9 @@ export const DAILY_METHOD_GET_CAMERA_FACING_MODE = 'get-camera-facing-mode';
 export const DAILY_METHOD_APP_MSG = 'app-msg';
 export const DAILY_METHOD_ADD_FAKE_PARTICIPANT = 'add-fake-participant';
 export const DAILY_METHOD_SET_SHOW_NAMES = 'set-show-names';
+export const DAILY_METHOD_SET_SHOW_LOCAL_VIDEO = 'set-local-video-display';
+export const DAILY_METHOD_SET_SHOW_PARTICIPANTS_BAR =
+  'set-participants-bar-display';
```

Comment:
is there a reason you chose not to match the string to the const (ie. `set-show-local-video` and `set-show-participants-bar`). If not, matching will make it easier to debug.

Reply:
I think I updated them at different times and they ended up out of sync. I'll change them--thanks for noticing!